deploy/olm-catalog/ibm-monitoring-grafana-operator/1.11.0/ibm-monitoring-grafana-operator.v1.11.0.clusterserviceversion.yaml

```diff
@@ -480,4 +480,5 @@ spec:
       name: Dong Wang
   provider:
     name: IBM
+  replaces: ibm-monitoring-grafana-operator.v1.10.4
```

Comment:
We do NOT want to add a `replaces` statement to the `1.11.0` CSV.  That was the one we specifcally asked squads to remove since the 1.11.0 version will be the base of the new `v3` channel.

We ONLY want the replaces added to the new `1.11.1` CSV to indicate it is a replacement of the `1.11.0` version.

Reply:
@Tirumalavasa Yes,  it should be 1.11.1 to replace 1.11.0. V3 and v1 are two different stream here.

Reply:
And when a specific version is under release protection, please do not change anything in CSV file. We are not supposed to change anything released which may raise some very weird bug.

Reply:
@dongwangdw Understood, i am reverting this change.

Reply:
@dongwangdw  can you verify the PR now.